app/helpers/application_helper.rb

```diff
@@ -364,4 +364,17 @@ def has_out_of_date_configuration?(service)
     return unless service
     service.pending_affecting_changes?
   end
+
+  def rails_asset_url_tag
+    javascript_tag("window.rails_asset_host = '#{rails_asset_host_url}'", { type: "text/javascript"})
+  end
+
+  def rails_asset_host_url
+    asset_host_enabled = Rails.configuration.asset_host.present?
```

Comment:
Not sure I get it - there are two `asset_host` values in different places in config tree? :thinking:

Reply:
> If I understand correctly Rails.configuration.asset_host is coming from config/environments/{env}.rb, and Rails.configuration.three_scale.asset_host comes from settings.yml.

Yes.

> From the change and the comment in webpack config, it seems to me that we always assume that this asset_host should be empty on compilation time?

Yes.

> If so, shouldn't we force it to be empty, so we don't have double-host issues?

Do you mean force it empty during compilation time? It's the same @akostadinov said. Yes, I agree.

Reply:
I added a callback that returns an error during assets compilation if the asset host is set. I couldn't find an easy way to just unset the asset host transparently. That would need more time but I can try if you guys think it's needed.